packages/apps-config/src/api/spec/uart.ts

```diff
@@ -21,17 +21,20 @@ const definitions: OverrideBundleDefinition = {
         Value: 'Vec<u8>',
         ReasonIndex: 'u32',
         LottoIndex: 'u32',
+        MemberId: 'u64',
+        ProposalId: 'u64',
+        TokenBalance: 'Balance',
         LotteryKind: {
-          _enum: {
-            Routine: 'Null',
-            TreasuryFunded: 'ReasonIndex'
-          }
+          _enum: [
+            'Routine',
+            'TreasuryFunded(ReasonIndex)'
```

Comment:
The original enum here was correct, `TreasuryFunded(ReasonIndex)` is not a valid type.